### ISSUE TITLE
Strict Standards Error

### ISSUE THREAD

**Issue by JamesGeiger** [Automattic/table-of-contents#2]
Hi Automatticians,

I am hoping to use this plugin, but I am getting "Non-static method Table_Of_Contents::init() should not be called statically" when I add it as a plugin and activate it. Any idea why?

**Comment by gedex** [comment]
```
Non-static method Table_Of_Contents::init() should not be called statically
```

Methods are called statically but missing `static` keyword. Adding `static` keyword to all methods in the class should solves the issue.

@JamesGeiger PR is always welcome.

**Comment by trepmal** [comment]
resolved in 616ad976f1c665fe0c5fe6f19af74d00768a775d
